### DISCUSSION TITLE
Is there a case where the user icon is applied as a 3D animation that a person walks?

### DISCUSSION THREAD

**Discussion by minkyumdev** [rnmapbox/maps#2523] [Q&A] [answered]
Is there a case where the user icon is applied as a 3D animation that a person walks?

Not found in rmmapbox document.

Please save me.

**Comment by Cmoen11** [comment, marked as answer]
You can change it to be a gif of someone walking... If that's the question?

**Reply by minkyumdev** [reply]
May I know how you applied it?

**Reply by Cmoen11** [reply]
https://github.com/rnmapbox/maps/blob/main/docs/UserLocation.md Look at the props, the children can be a custom icon.
